charts/topolvm/CHANGELOG.md

```diff
@@ -7,6 +7,21 @@ This file itself is based on [Keep a CHANGELOG](https://keepachangelog.com/en/0.
 
 ## [Unreleased]
 
+## [4.0.0] - 2022-01-07
+
+### Changed
+- remove k8s version specification from Chart.yaml (#403)
+- Make kubelet work directory overridable via single chart parameter (#410)
+  - **BREAKING**: The `node.kubeletWorkDirectory` parameter has been added, and the default values of other parameters regarding the host path have changed. Please review the settings related to the host path.
+- skip PDB when topolvm-scheduler isn't enabled (#417)
+
+### Added
+- Add topolvm-controller CLI flag to skip node finalize (#409)
```

Comment:
I assume it is related to topolvm itself rather than helm.

Reply:
This PR contains the changes related to helm, so I also added this item to the CHANGELOG of helm.

Reply:
Understood.